manageiq-gems-pending.gemspec

```diff
@@ -51,7 +51,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "nokogiri",                "~>1.6.8"
   s.add_runtime_dependency "openscap",                "~>0.4.3"
   s.add_runtime_dependency "ovirt",                   "~>0.15.1"
-  s.add_runtime_dependency "parallel",                "~>1.9" # For OvirtInventory
```

Comment:
@bdunne Even though this is being removed and the comment says "For OvirtInventory", can you please verify this is needed by anyone else that was getting it automatically?  azure comes to mind, and also the core repo, but I'm not sure what else.